BentoKit/BentoKit/Components/Toggle.swift

```diff
@@ -137,14 +137,17 @@ extension Component.Toggle {
 extension Component.Toggle {
     public final class StyleSheet: BaseViewStyleSheet<View> {
         public let text: LabelStyleSheet
+        public let spacing: CGFloat
 
-        public init(text: LabelStyleSheet) {
+        public init(text: LabelStyleSheet, spacing: CGFloat = 9) {
```

Comment:
Why 9?

Reply:
9 is the [default value currently being used](https://github.com/Babylonpartners/Bento/blob/c81f6ccb5df2cc61d80df3bc62728ea3c32b4c1f/BentoKit/BentoKit/Components/Toggle.swift#L105).

Reply:
But the `9` is still there, it's not being read from `spacing`.

Reply:
We're overriding this value later, essentially, when we run:

Reply:
I see. 👍

Reply:
If you want i can look into alternatives to prevent this, of course